.github/workflows/bench.yml

```diff
@@ -54,6 +62,12 @@ jobs:
 
           git add -A benchmark_results
           git add README.md
+
+          cd pages
+          trunk --config Trunk.prod.toml build --release --filehash=false
+          cd ..
+          git add -A docs
```

Comment:
Instead of adding the built app to the repository, I usually use the `peaceiris/actions-gh-pages@v3` action to deploy a directory directly to a `gh-pages` branch. [Here's how rkyv does it](https://github.com/rkyv/rkyv/blob/master/.github/workflows/gh-pages.yml#L21-L25). This allows us to avoid checking in binaries to the main branch.